### PR TITLE
Fixing bug #145. 

### PR DESCRIPTION
**Pull request by kshyju**
Fix for the bug #145 . Checking to see whether the collection has at least one item before making the Last() method call on the collection. Thanks!

### CODE REVIEW

File: Opserver/Views/SQL/Operations.Top.Detail.cshtml [hunk 1]
```diff
@@ -159,14 +159,14 @@
...
-                            var lastStatement = plan.Statements.Last();
+                            var lastStatement = plan.Statements.Any() ? plan.Statements.Last() : null;
                             var textPos = 0;
                         }
                         @foreach (var statement in plan.Statements)
                         {
                             <pre class="pre-code prettyprint lang-sql @(textPos == op.StatementStartOffset && plan.Statements.Count > 1 ? " selected-statement" : "")">@statement.ParameterDeclareStatement
 @if (!truncated){@statement.StatementTextWithoutInitialParams}
-@if(truncated && statement == lastStatement){@op.FullText}</pre>
+@if(truncated && lastStatement!=null && statement == lastStatement){@op.FullText}</pre>
```
Comment (NickCraver) [inline Opserver/Views/SQL/Operations.Top.Detail.cshtml]: What's this null check for - wouldn't we not loop if there are none? :)

File: Opserver/Views/SQL/Operations.Top.Detail.cshtml [hunk 2]
```diff
@@ -159,14 +159,14 @@
                     <div>
                         @{
                             var truncated = plan.Statements.Any(st => st.StatementText != null && st.StatementText.Length > 3998);
-                            var lastStatement = plan.Statements.Last();
+                            var lastStatement = plan.Statements.Any() ? plan.Statements.Last() : null;
```
Comment (NickCraver) [inline Opserver/Views/SQL/Operations.Top.Detail.cshtml]: `.LastOrDefault()`

Comment (kshyju) [inline Opserver/Views/SQL/Operations.Top.Detail.cshtml]: Ah ! I missed it. Thank you. `LastOrDefault()` wins. Ignore the method name in my commit message :). I am still recovering from the depression caused by #130

**Comment by NickCraver** [comment]
Thanks!

**Pull request merged**